docs/index.md

```diff
@@ -25,6 +25,7 @@ api/data_collector
 :glob:
 :caption: Tutorials
 tutorials/**/index
+tutorials/behavioral_cloning
```

Comment:
I think we can include this tutorial in a new tab called "Using Datasets". What do you think?

Reply:
Sounds good, done